### DISCUSSION TITLE
Aplicabilidade do sistema em jogos

### DISCUSSION THREAD

**Discussion by JoseSoares01** [viscerae/optimaldnd#3] [Ideas]
Estive de olho neste projeto e é bastante interessante, as farramentas estão 100% funcionais, porém há uma questão quanto a aplicabilidade da ferramente em um modelo mobile, com o objetivo de simplificar e ajustar a comodidade, é possivel?

**Comment by viscerae** [comment]
O projeto vai estar dividido em várias fases, sendo que a primeira delas será desenvolvimento das ferramentes e do site por completo, após isso irei começar a trabalhar na compatibilidade tanto para Iphone como para Android mas primeiro quero assegurar-me que tudo está funcional para utilização do publico.

Ou seja, será implementado mas muito mais para a frente. Conto com a vossa colaboração para melhorar e a desenvolver uma ferramenta open-source e gratuita de auxilio em sessões de DnD, juntos conseguimos por um fim a ferramentas que exigem pagamentos e subscrições que (segundo a minha opinião) neste caso não são necessárias.

Obrigado pela sugestão, irei adicionar ao plano de desenvolvimento do projeto!
👍🏻
________________________

✧ Viscera ✧
